apps/docs.blocksense.network/components/common/RadioGroup.tsx

```diff
@@ -0,0 +1,124 @@
+'use client';
+
+import React, {
+  createContext,
+  useContext,
+  ReactNode,
+  HTMLAttributes,
+  FieldsetHTMLAttributes,
+} from 'react';
+
+import { cn } from '@blocksense/ui/utils';
+import { Label } from '@blocksense/ui/Label';
+import { Input } from '@blocksense/ui/Input';
+
+type RadioGroupContextValue = {
+  value: string;
+  name: string;
+  onChange: (value: string) => void;
+};
+
+const RadioGroupContext = createContext<RadioGroupContextValue>(
+  {} as RadioGroupContextValue,
+);
+
+type RadioGroupProps = Omit<
+  FieldsetHTMLAttributes<HTMLFieldSetElement>,
+  'onChange'
+> & {
```

Comment:
```suggestion
type RadioGroupProps = FieldsetHTMLAttributes<HTMLFieldSetElement> & {
```

You can remove `Omit` and rename `onChange` prop to `onValueChange`.